modules/client-configuration/pages/clients-opensuseleap.adoc

```diff
@@ -68,14 +76,14 @@ ifeval::[{uyuni-content} == true]
 .{opensuse} Channels - CLI
 |===
 
-| OS Version               | {opensuse} Leap 15.6               | {opensuse} Leap 15.5               | {opensuse} Leap 15.4
-| Base Channel             | opensuse_leap15_6                  | opensuse_leap15_5                  | opensuse_leap15_4
-| Client Channel           | opensuse_leap15_6-uyuni-client     | opensuse_leap15_5-uyuni-client     | opensuse_leap15_4-uyuni-client
-| Updates Channel          | opensuse_leap15_6-updates          | opensuse_leap15_5-updates          | opensuse_leap15_4-updates
-| Non-OSS Channel          | opensuse_leap15_6-non-oss          | opensuse_leap15_5-non-oss          | opensuse_leap15_4-non-oss
-| Non-OSS Updates Channel  | opensuse_leap15_6-non-oss-updates  | opensuse_leap15_5-non-oss-updates  | opensuse_leap15_4-non-oss-updates
-| Backports Updates Channel| opensuse_leap15_6-backports-updates| opensuse_leap15_5-backports-updates| opensuse_leap15_4-backports-updates
-| SLE Updates Channel      | opensuse_leap15_6-sle-updates      | opensuse_leap15_5-sle-updates      | opensuse_leap15_4-sle-updates
+| OS Version               | {opensuse} Leap 16.0               | {opensuse} Leap 15.6               | {opensuse} Leap 15.5               | {opensuse} Leap 15.4
+| Base Channel             | opensuse_leap16_0                  | opensuse_leap15_6                  | opensuse_leap15_5                  | opensuse_leap15_4
```

Comment:
As of today, there's a single channel for the OS, that gets the update.

And it's quite likely it will stay that way, also quite likely there will be no SLE Updates channel, as [openSUSE Leap Micro 6.1 will not have it already](https://news.opensuse.org/2024/11/20/leap-micro-61-alpha-availability/), and openSUSE Leap Micro 6.1 is based on the SLFO as openSUSE Leap 16.0.